### PR TITLE
Created a readme file for installing and configuring tmuxinator

### PR DESCRIPTION
**Pull request by shubh-agrawal**
Useful in executing commands either in sequential order or simultaneously in different terminal windows or panes

### CODE REVIEW

File: tmuxinator.md [hunk 1]
```diff
@@ -0,0 +1,200 @@
+# tmuxinator
+###Executing commands sequentially in different panes and windows
+
+Open terminal
+
+$ sudo apt-get install tmux
+
+$ sudo apt-get install ruby
+
+$ sudo su
```
Comment (abinashmeher999) [inline tmuxinator.md]: Why do you need to change to root?

File: tmuxinator.md [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+
+$ gem install tmuxinator
```
Comment (abinashmeher999) [inline tmuxinator.md]: If you are changing to root just to install the tmuxinator gem with root priviledges, you can also do `sudo -E gem install tmuxinator`

**Comment by abinashmeher999** [comment]
Nice that you created a documentation for this one!

**Comment by shivamvats** [comment]
Good job @shubh-agrawal !

**Comment by aranyadan** [comment]
sudo su is not required i did everything with sudo -E and it works.

**Comment by shubh-agrawal** [comment]
@abinashmeher999  "sudo -E " will surely do the same job . Whatever I have written is the process through which I underwent through. That command needed root previleges , so shall I change the documentation or retain it?

**Comment by abinashmeher999** [comment]
Changing to super user shouldn't be done unless it is absolutely necessary, like when you conclude there isn't any other way this can be done. It has many disadvantages and I need not remind you what a headache the controls module has caused because of the requirement to change to superuser. In this case however, it is needed so my suggestion is to remove redundant steps. Change the documentation to `sudo -E` because that directly conveys your intention rather then changing to su and then changing back.

**Comment by shubh-agrawal** [comment]
changes made!Please comment if more changes are required

**Comment by shivamvats** [comment]
+1 from my side.

**Pull request merged**